### PR TITLE
[Student][E2E][MBL-13002]: E2E test for grades page

### PR DESCRIPTION
**Pull request by joehoag**
Also fixed several a11y issues.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1009045&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/student/src/androidTest/java/com/instructure/student/ui/e2e/QuizzesE2ETest.kt [hunk 1]
```diff
@@ -140,30 +141,4 @@ class QuizzesE2ETest: StudentTest() {
...
-
-        QuizzesApi.publishQuiz(
-                courseId = courseId,
-                quizId = result.id,
-                teacherToken = teacherToken,
-                published = true
-        )
-
-        return result
-    }
```
Comment (joehoag) [inline apps/student/src/androidTest/java/com/instructure/student/ui/e2e/QuizzesE2ETest.kt]: Moved this method to QuizzesApi.kt

File: apps/student/src/main/res/layout/dialog_what_if.xml [hunk 2]
```diff
@@ -22,6 +22,14 @@
...
     android:gravity="center_horizontal">
 
+    <!-- provide speakable text for EditText below -->
+    <TextView
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:visibility="gone"
+        android:text="@string/enterWhatIfScore"
+        android:labelFor="@id/currentScore" />
+
```
Comment (joehoag) [inline apps/student/src/main/res/layout/dialog_what_if.xml]: A11y fix to add speakable text to currentScore element.

File: apps/student/src/main/res/layout/fragment_course_grades.xml [hunk 3]
```diff
@@ -137,6 +140,8 @@
                         style="@style/TextStyle.Tertiary.Dark"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
+                        android:minHeight="48dp"
+                        android:minWidth="48dp"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/fragment_course_grades.xml]: A11y touch-target-size fixes for checkbox areas "Base on graded assignments" and "Show What-If Score"

File: apps/student/src/main/res/layout/viewholder_grade.xml [hunk 4]
```diff
@@ -56,10 +56,10 @@
     <ImageView
         android:id="@+id/edit"
         style="@style/AdapterItemIcon"
-        android:layout_width="42dp"
-        android:layout_height="42dp"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
         android:contentDescription="@string/edit"
-        android:padding="8dp"
+        android:padding="11dp"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/viewholder_grade.xml]: A11y touch-target-size fix for pencil/edit icon that gets shown in "What-If" scenario.

**Pull request merged**